data_model.md

```diff
@@ -0,0 +1,399 @@
+---
+layout: datamodel
+permalink: /datamodel/
+---
+
+Data Model
+==========
+{: .anchor}
+The goal of this document is to explain the Solar Forecast Arbiter data
+model. To keep the framework architecture simple and secure, the API
+closely follows this data model. Therefore, it is worthwhile for API
+users to understand the data model. Please see
+[dev-api.solarforecastarbiter.org](https://dev-api.solarforecastarbiter.org/)
+for the technical API documentation. The Dashboard should be clear
```

Comment:
@lboeman can you make an issue to remind us to link to the dashboard intro once that doc is also posted here?

Reply:
#50